src/main/java/com/flipkart/ranger/finder/CuratorSourceConfig.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2015 Flipkart Internet Pvt. Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.flipkart.ranger.finder;
+
+import com.flipkart.ranger.model.Deserializer;
+
+public class CuratorSourceConfig<T> extends SourceConfig{
+    private String connectionString;
+    private String namespace;
+    private Deserializer<T> deserializer;
```

Comment:
Since this is only for handling ZK Specific data, we should rename it ZookeeprNodeDataDecoder